content/techniques/logger.md

```diff
@@ -35,6 +35,8 @@ await app.listen(3000);
 
 Values in the array can be any combination of `'log'`, `'error'`, `'warn'`, `'debug'`, and `'verbose'`.
 
+To disable color in the default logger's messages, set the `NO_COLOR` environment variable.
```

Comment:
Can we turn this into a blockquote HINT? (you can find examples of how to use it in other chapters)

Reply:
Updated to blockquote HINT as per suggestion.